sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -1557,6 +1557,17 @@ class PlanResolutionSuite extends AnalysisTest {
     checkFailure("testcat.tab", "foo")
   }
 
+  test("SPARK-32356: forbid null type in create view") {
+    val sql1 = "create view v as select null as c"
+    val sql2 = "alter view v as select null as c"
```

Comment:
can we test temp view as well? also the df api `df.createTempView`

Reply:
added.